### PR TITLE
Add dotnet labels-projects doc

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes https://github.com/dotnet/docs/issues/17575

* Move labels and projects doc to contrib guide (after this is merged I'll replace the content in the file in the styleguide folder with a link to here).
* Consolidate two docs that had similar names in the TOC ("Contribute to .NET docs" and "How to contribute to .NET docs").

@BillWagner please review.

### CODE REVIEW

File: Contribute/dotnet/labels-projects.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+- WPF
+
+### Releases
+
+![:checkered_flag: Release: on dark yellow](./media/labels-projects/release.png "Prefix for release labels")
+
+Issues tagged for a specific release are noted with the `:checkered_flag: Release:` prefix and have a dark yellow background.
+
+- .NET Core 2.2
+- .NET Core 3.0
```
Comment (BillWagner) [inline Contribute/dotnet/labels-projects.md:103]: We've added the following now:

```suggestion
- .NET Core 3.0
- .NET Framework 4.8
- .NET 5
```

File: Contribute/dotnet/labels-projects.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+### Priority
+
+Priority labels are all `P` followed by a single digit. Lower numbers are higher priority:
+
+- P0
+- P1
+- P2
```
Comment (BillWagner) [inline Contribute/dotnet/labels-projects.md:111]: We've added a "P3":

```suggestion
- P2
- P3
```

File: Contribute/dotnet/labels-projects.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+### What about the other labels?
+
+There are many other labels used by the content teams to manage different classifications of issues. If you're not on the content team, you can ignore these other labels.
+
+## Projects
+
+Contributors should check the [Projects for .NET community collaborators](https://github.com/dotnet/docs/projects/35). We've created different columns for maintenance, document updates, and new content tasks. This can be a way to find tasks of interest. (Note that the project view can be filtered using the labels described above.)
```
Comment (BillWagner) [inline Contribute/dotnet/labels-projects.md:119]: The paragraph should be deleted now. We've closed this project.

**Comment by tdykstra** [comment]
@BillWagner Thanks for reviewing!

**Comment by tdykstra** [comment]
#sign-off

**Comment by tdykstra** [comment]
@Jim-Parker This is ready to merge.

**Pull request merged**